### DISCUSSION TITLE
场景内纯lua实现的mmo，单个场景只能支持到200个玩家

### DISCUSSION THREAD

**Discussion by ghost** [cloudwu/skynet#1814] [Q&A]
是不是还有很大的提升空间？大家做的单场景能到多少，纯lua实现的场景。

**Comment by terry8210** [comment]
主要看业务复杂度？是不是有些逻辑可以提到独立服务处理，例如AOI，寻路等。还是具体情况具体分析吧，这样一句话话很难说单个VM的支撑能力。

**Reply by ghost** [reply]
基本上是最基础的那种mmo，我发现的问题主要是我把很多东西都放在场景里了。

**Reply by ghost** [reply]
除了aoi，其他都放在场景服务里了。

**Comment by sniper00** [comment]
> 是不是还有很大的提升空间？大家做的单场景能到多少，纯lua实现的场景。

能做到2000人
